docs/src/modules/components/MarkdownDocs.js

```diff
@@ -194,12 +193,14 @@ MarkdownDocs.propTypes = {
   disableAd: PropTypes.bool,
   disableToc: PropTypes.bool,
   markdown: PropTypes.string,
-  // You can define the direction location of the markdown file.
-  // Otherwise, we try to determine it with an heuristic.
-  markdownLocation: PropTypes.string,
   req: PropTypes.func,
   reqPrefix: PropTypes.string,
   reqSource: PropTypes.func,
 };
 
+if (process.env.NODE_ENV !== 'production') {
+  // eslint-disable-next-line no-useless-concat, react/forbid-foreign-prop-types
+  MarkdownDocs['propTypes' + ''] = exactProp(MarkdownDocs.propTypes);
```

Comment:
Assuming we don't need to run react-docgen with this component.

```suggestion
  MarkdownDocs.propTypes = exactProp(MarkdownDocs.propTypes);
```

Reply:
Nice. I always assumed this was to prevent issues with babel-strip-proptypes.

Reply:
Hum, I haven't thought about this one. I guess we will see 😁

Reply:
Seems to work